docs/install-function-mesh.md

```diff
@@ -66,6 +66,20 @@ This example shows how to install Function Mesh through [Helm](https://helm.sh/)
 
    2. Install Function Mesh Operator.
 
+        There are some parameters for the Function Mesh Operator, and you can customize them for a particular purpose. 
+        The following table lists the configurable parameters of the operator and their default values.
+        | Parameters | Description | Default|
+        |--         |--           |--       |
+        |enable-leader-election| Whether contorller-manager should enable leader-election| true|
```

Comment:
In my opinion, we should use `function-mesh-controller-manager` to replace `Function Mesh Controller Manager ` because the name of deployment is `function-mesh-controller-manager` and the parameters are the config of this deployment.
<img width="586" alt="image" src="https://user-images.githubusercontent.com/7600925/161248656-56240a8a-45a2-4e30-9def-cc28b451f2b6.png">